kubectl-cloudflow/docker/client.go

```diff
@@ -136,3 +137,50 @@ func imageMatchesNameAndTag(image types.ImageSummary, imageNameAndTag string) bo
 	}
 	return false
 }
+
+// GetCloudflowStreamletDescriptorsForImage extracts the configuration of a Cloudflow label from a docker image
+// and the image with digest in a struct
+func GetCloudflowStreamletDescriptorsForImage(cli *client.Client, imageName string) (domain.CloudflowStreamletDescriptorsDigestPair, string) {
```

Comment:
it would be great if we could get the label from the image without actually pulling the image. (That might be an improvement we can do later, after this PR, but if it is easy, rather do it right away?)

Reply:
will check ..